utils/build/webpack.base.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const packageManifest = require(path.resolve('package.json'));
+
+const options = {
+  mode: 'production',
+  entry: path.resolve('src', 'index.js'),
+  devtool: 'source-map',
+  optimization: {
+    minimizer: [new OptimizeCSSAssetsPlugin({})]
+  },
+  plugins: [
+    /**
+     * Allows the `data-garden-version` attribute to be applied to all
+     * views without importing entire package.json into the bundle
+     */
+    new webpack.DefinePlugin({
+      PACKAGE_VERSION: JSON.stringify(packageManifest.version)
+    }),
+    new webpack.BannerPlugin(`Copyright Zendesk, Inc.
```

Comment:
You could use `trim` and format this a little nicer.

```js
    new webpack.BannerPlugin(`
Copyright Zendesk, Inc.

Use of this source code is governed under the Apache License, Version 2.0
found at http://www.apache.org/licenses/LICENSE-2.0
    `.trim()),
```

Reply:
Fixed.